src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { PopupPlacement } from '../../constants';
+import PopupBase from '../PopupBase';
+import styles from './Tooltip.scss';
+import bem from '../../utils/bem';
+
+const { elem } = bem('Tooltip', styles);
+
+interface Props {
+    children: NonEmptySingleReactNode;
+    content: NonEmptySingleReactNode;
+    placement?: PopupPlacement;
+}
+
+const Tooltip: React.FC<Props> = props => {
+    const { placement, content, children } = props;
+
+    const createMouseEnterHandler = setPopupVisibility => () => {
+        setPopupVisibility(true);
+    };
+
+    const createMouseLeaveHandler = setPopupVisibility => () => {
+        setPopupVisibility(false);
+    };
+
+    const renderAnchor = ({ setPopupVisibility }) => (
+        <div
+            onMouseEnter={createMouseEnterHandler(setPopupVisibility)}
+            onMouseLeave={createMouseLeaveHandler(setPopupVisibility)}
+        >
+            {children}
+        </div>
+    );
+
+    const renderPopup = () => (
+        <div>
+            <div {...elem('container', props)}>
+                {content}
+                <div {...elem('arrow', props)} />
+            </div>
+        </div>
```

Comment:
Can we get rid of div wrapper? I tried to delete it locally, but this change brokes `Tooltip`. This div applies styles `top: 0, left: 0` that go probably from  `PopperJS` and when I deleted  `top: 0, left: 0`  popup appears correctly. That fixed `Edge` and `IE11` as well.
![Screenshot 2019-12-06 at 11 25 33](https://user-images.githubusercontent.com/19436304/70319202-04667680-1822-11ea-8c8b-9051f0ce3dcf.png)

![Screenshot 2019-12-06 at 12 06 09](https://user-images.githubusercontent.com/19436304/70319184-fadd0e80-1821-11ea-94fe-434880cb2563.png)